secure-biometric-interface-integrator/lib/standardConstant.js

```diff
@@ -0,0 +1,77 @@
+const states = {
+  LOADING: "LOADING",
+  LOADED: "LOADED",
+  ERROR: "ERROR",
+  AUTHENTICATING: "AUTHENTICATING",
+};
+
+const DeviceStateStatus = {
+  Ready: "READY",
+  "Not Ready": "NOTREADY",
+  Busy: "BUSY",
+  "Not Registered": "NOTREGISTERED",
+};
+
+const DeviceState = {
+  READY: {
+    value: "READY",
+    name: "Ready",
+    class: "ready",
+    symbol: "\u25CF",
+  },
+  NOTREADY: {
+    value: "NOTREADY",
+    name: "Not Ready",
+    class: "not-ready",
+    symbol: "\u25CF",
+  },
+  BUSY: {
+    value: "BUSY",
+    name: "Busy",
+    class: "busy",
+    symbol: "\u25CF",
+  },
+  NOTREGISTERED: {
+    value: "NOTREGISTERED",
+    name: "Not Registered",
+    class: "not-registered",
+    symbol: "\u25CE",
```

Comment:
have a comment here that its for the bulleye?

Reply:
hi @gsasikumar , commented on all of the symbol, check this [044194a](https://github.com/mosip/esignet-plugins/pull/5/commits/044194a5882341fd950e519ef9b08f59144e6caa) commit